.github/workflows/pr.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Check version
         run: yarn ts-node ci/version-check
 
-      - name: E2E tests
+      - name: Run E2E tests
         run: |
           NX_PLUGIN_DIR=$(pwd)
```

Comment:
Can we use the local-test script?

Reply:
Good idea.  Will make a ticket.

Reply:
Created as https://github.com/DigitalInnovation/onyx-nx/issues/2156.